setup.sh

```diff
@@ -134,6 +134,28 @@ if [[ $DEVICE == "tpu" ]]; then
     fi
 fi
 
+EXTRA_PIP_INSTALL_ARGS=()
+if [ "$MODE" != 'stable' ]; then
+  EXTRA_PIP_INSTALL_ARGS+=(--no-deps)
+fi
+
+if [ -f 'requirements-pyproj.txt' ]; then
+  python3 -m uv pip install --resolution=lowest -r requirements-pyproj.txt
+elif [ -f "${MAXTEXT_REPO_ROOT}"'/requirements-pyproj.txt' ]; then
+  python3 -m uv pip install -r "${MAXTEXT_REPO_ROOT}"'/requirements-pyproj.txt'
+else
+  >&2 printf 'Could not find requirements-pyproj.txt, looked in ("%s", "%s")\n' \
+      "${PWD:-$(pwd)}" \
+      "${MAXTEXT_REPO_ROOT}"
+  exit 2
+fi
+
+# Install maxtext package
+if [ -f 'pyproject.toml' ]; then
+  python3 -m uv pip install -e . "${EXTRA_PIP_INSTALL_ARGS[@]}" --resolution=lowest
+  install_maxtext_github_deps
+fi
```

Comment:
Won't the pip installation run twice when `bash setup.sh` is run locally?

Reply:
I don't see the other place in the codebase `pip install -e` is done on maxtext other than the setup.sh. The Dockerfiles aren't doing it.